packages/core/types/FsdtMessage.ts

```diff
@@ -10,8 +11,9 @@ export type FsdtServerMessage = FsdtMessage & {
 }
 
 export type FsdtLogMessage = {
```

Comment:
Missing source ?

Reply:
Only the FsdtServerMessage has the field "source"

Reply:
Ok, I think I miss understood how it works, mb